RELEASE_NOTES.txt

```diff
@@ -2,6 +2,238 @@ Apache Ignite Release Notes
 ===========================
 
 
+Apache Ignite In-Memory Distributed Database 2.10.0
+-----------------------------------------------------------
+
+
+(!) WARNINGS:
+* The community accepted MVCC API discontinuation in future releases
+* Enabling/disabling WAL doesn't work properly on unstable topology and will be fixed soon
+* Spring Data moved to ignite-extension and will be released separately
+* Node.js, Python, PHP thin clients moved to a dedicated repository
+
+
+Ignite:
+* Added API for checking network connectivity between all nodes in a cluster
+* Added API to get segment sizes
+* Added InitialSize and MaxSize to DataRegion metrics
+* Added RebalancingPartitionsTotal metrics
+* Added TDE CLI management tool
+* Added TDE cache key rotation
+* Added a metric for processed keys when rebuilding indexes
+* Added an ability to check driver metrics in Cassandra store
+* Added cluster performance profiling
+* Added cluster performance tool to build cluster performance report
```

Comment:
Should be excluded (this is feature from the`ignite-extensions` module)

Reply:
Fixed.